pkgs/top-level/perl-packages.nix

```
@@ -10400,6 +10400,9 @@ let
     };
     propagatedBuildInputs = [ ArchiveZip DBFile FileWhich IOString ImageSize JSONXS LWP ParseRecDescent TextUnidecode URI XMLLibXML XMLLibXSLT shortenPerlShebang ];
```

`perl 5.32.0` was recently merged to master, which no longer has  `Pod::Parser` in core. 

```suggestion
    propagatedBuildInputs = [ ArchiveZip DBFile FileWhich IOString ImageSize JSONXS LWP ParseRecDescent PodParser TextUnidecode URI XMLLibXML XMLLibXSLT shortenPerlShebang ];
```

---

it is done in #97098

---

@volth Ok, would it be best to wait until #97098 hits master in that case?

---

or merge this to staging

---

@stigtsp @volth do you need me to do something in this PR?